### PR TITLE
Added Asset Info Popover to Camera Feed in Consultation Page

### LINKED ISSUE #8399: Add asset info pop-up for camera feed in consultation page
**Is your feature request related to a problem? Please describe.**

Currently, we are showing the middleware and IP address of a vital monitor in the consultation page, but the camera feed doesn't have that info in the consultation page, which can be a good add-on towards the UX

**Describe the solution you'd like**

- [ ] Add the asset info pop-up similar to the vital monitor to the camera feed page to show only IP address and hostname
- [ ] make the pop-up in the vital monitor page a reusable component

**Screenshots**

image
image

### PR DESCRIPTION
**Pull request by Jacobjeevan**
## Proposed Changes

- Fixes #8399

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/MonitorAssetPopover.tsx [hunk 1]
```diff
@@ -51,13 +53,17 @@ const VitalsMonitorAssetPopover = ({
               <p>{asset?.name}</p>
             </div>
             <div className="flex flex-col gap-1">
-              <p className="text-sm md:text-base">Middleware Hostname:</p>
+              <p className="text-sm text-black md:text-base">
+                Middleware Hostname:
+              </p>
```
Comment (rithviknishad) [inline src/Components/Common/MonitorAssetPopover.tsx]: Can we add this to locale and use `useTranslations`?

File: src/Components/Common/MonitorAssetPopover.tsx [hunk 2]
```diff
@@ -51,13 +53,17 @@ const VitalsMonitorAssetPopover = ({
...
               <p className="break-words text-secondary-600">
                 {asset?.resolved_middleware?.hostname}
               </p>
             </div>
             <div className="flex flex-col gap-1">
-              <p className="text-sm md:text-base">Local IP Address:</p>
+              <p className="text-sm text-black md:text-base">
+                Local IP Address:
+              </p>
```
Comment (rithviknishad) [inline src/Components/Common/MonitorAssetPopover.tsx]: same for here and other places where there are representable hard-coded texts in this file.

**Review by rithviknishad — COMMENTED** [review]
LGTM, just a minor suggestion

File: src/Components/Common/MonitorAssetPopover.tsx (hunk 1 above)
Comment (Jacobjeevan) [reply src/Components/Common/MonitorAssetPopover.tsx]: Sure, added them. I used google translate, so not sure how exact the translations are for Hindi, Kannada and Tamil.

Comment (rithviknishad) [reply src/Components/Common/MonitorAssetPopover.tsx]: We use crowdin for translations for other languages, so usually in PR's we only add for english and crowdin should take care of other languages. 

cc: @bodhish

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Pull request merged**